tests/portalbackend/PortalBackend.DBAccess.Tests/ConnectorRepositoryTests.cs

```diff
@@ -442,7 +442,10 @@ public async Task GetConnectorUpdateInformation_ReturnIsProviderTrue()
 
         // Assert
         result.Should().NotBeNull();
-        result?.IsProviderCompany.Should().BeTrue();
+        if (result != null)
+        {
+            result.IsProviderCompany.Should().BeTrue();
+        }
```

Comment:
the null check is redundant as there's a NotBeNull assert right above. As this is a unit-test the use of '!' operator is more than justified in such cases.
```suggestion
        result!.IsProviderCompany.Should().BeTrue();
```

Reply:
Thanks @ntruchsess ,Fixed